lib/src/cupertino/cupertino_controls.dart

```diff
@@ -443,6 +443,7 @@ class _CupertinoControlsState extends State<CupertinoControls>
           playing: controller.value.isPlaying,
           playIcon: widget.playButtonIcon,
           pauseIcon: widget.pauseButtonIcon,
+          size: 14,
```

Comment:
```suggestion
          size: widget.size,
```
?

Reply:
size was set to affect both play button icon and bottom bar icon. We needed different sizes and it has to match other icons on the bottom bar